docker/indexer.Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM python:3.9-slim
```

Comment:
I think we can revert back to `python:3.9`. In a previous PR I made a change to use `python:3.9-slim`: https://github.com/regen-network/regen-web/pull/2125

Today I ran into the same issue I initially reported. But doing another `docker pull python:3.9` seems to have pulled a newer image that has fixed this issue. I believe this issue explains more why this was happening for me: https://github.com/docker-library/python/issues/837